specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/examples/AvailableWorkloadProfiles_Get.json

```diff
@@ -124,6 +124,19 @@
             "id": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/providers/Microsoft.App/availableManagedEnvironmentsWorkloadProfileTypes/CPUOptimized-3",
             "name": "CPUOptimized-3",
             "type": "Microsoft.App/availableManagedEnvironmentsWorkloadProfileTypes"
+          },
+          {
+            "location": "East US",
+            "properties": {
+              "displayName": "Consumption",
+              "billingMeterCategory": "ConsumptionBasedBilling",
+              "applicability": "Custom",
+              "cores": 16,
+              "memoryGiB": 32
+            },
```

Comment:
we do not have coreand memory for consumption, right?

Reply:
This is representing the max values for a container app in this workload profile